components/Ui/UiCard.vue

```diff
@@ -32,21 +32,21 @@
           <h5 v-if="subtitle">
             {{ subtitle }}
           </h5>
-        </div>
-        <div class="cds--row">
-          <div v-if="hasTags(tags)" class="card__tags">
-            <bx-tag
-              v-for="tag in tags"
-              :key="tag"
-              class="card__tag"
-              type="purple"
-            >
-              {{ tag }}
-            </bx-tag>
+          <div class="card__tags">
+            <template v-if="hasTags(tags)">
+              <bx-tag v-if="primaryTag" class="card__tag" type="purple">
+                {{ primaryTag }}
+              </bx-tag>
+              <bx-tag
+                v-for="tag in tags"
```

Comment:
I suggest to rename `tags` to `secondaryTags` to avoid confusion

Reply:
Updated in https://github.com/Qiskit/qiskit.org/pull/3442/commits/a1c9db3717e90bdfee51def9905d8adc7bca823e